src/flag_gems/__init__.py

```diff
@@ -301,7 +294,9 @@ def enable(
             ("zeros", zeros),
             ("zeros_like", zeros_like),
         ),
-        user_unused_ops_list=[] if unused is None else unused,
+        user_unused_ops_list=list(
+            set([] if unused is None else unused) & set(aten_patch_list)
+        ),
```

Comment:
Does it mean that operators in unused or aten_patch_list should be excluded from `enable`?

If so, then I think it should be the union of the two sets, rather than the intersection of them.